### ISSUE TITLE
Rotating object in SceneView leaves a visual trail

### ISSUE THREAD

**Issue by Wrakor** [ThomasGorisse/sceneform-android-sdk#15]
I'm loading the model like this:

```
ModelRenderable.builder()
            .setSource(this, Uri.parse("GroundVehicle.glb"))
            .setIsFilamentGltf(true)
            .build()
            .thenAccept(
                Consumer { modelRenderable: ModelRenderable ->
                    //Add anything you want to do here....
                    Log.d("Sceneform","Model loaded")

                    createNode(modelRenderable)
                })
            .exceptionally { throwable: Throwable? ->
                //Add anything you want to do here e.g.:
                Log.d("Sceneform", "ModelRenderable.builder exception: " + throwable)
                null
            }

private fun createNode(modelRenderable: ModelRenderable) {
        val transformationSystem = TransformationSystem(resources.displayMetrics, object: SelectionVisualizer {
            override fun applySelectionVisual(node: BaseTransformableNode?) {}

            override fun removeSelectionVisual(node: BaseTransformableNode?) {}
        })

        val transformableNode = TransformableNode(transformationSystem).apply {
            rotationController.isEnabled = true
            scaleController.isEnabled = false
            translationController.isEnabled = false
            setParent(viewBinding.sceneView.scene)
            //localPosition = Vector3(0f, 0f, -1f)
            localScale = Vector3(0.7f, 0.7f, 0.7f)
            renderable = modelRenderable // Build using CompletableFuture
            select()
        }

        viewBinding.sceneView.scene.addChild(transformableNode)

        viewBinding.sceneView.scene.addOnPeekTouchListener { hitTestResult, motionEvent ->
            transformationSystem.onTouch(hitTestResult, motionEvent)
        }

        viewBinding.sceneView.scene.camera.localPosition = Vector3(0f, 1f, 5f)
    }
```

which leaves a visual trail after rotating:

![image](https://user-images.githubusercontent.com/2973191/111514771-f48a5180-8749-11eb-9b09-c5ba84c9a417.png)

**Comment by ThomasGorisse** [comment]
What background are you using?

**Comment by Wrakor** [comment]
> What background are you using?

It's a solid color (#14)

**Comment by ThomasGorisse** [comment]
Fixed here: b6c6cee9d337922464e82c1b4e3819b85dd3462f
Please review the [SceneView Sample app](https://github.com/ThomasGorisse/sceneform-android-sdk/blob/38bcb379b569ebe079e8611c5bf1ef78749bcffe/samples/sceneview/src/main/java/com/google/ar/sceneform/samples/sceneview/MainActivity.java) for usage
Use the snapshot version as describe [here ](https://thomasgorisse.github.io/sceneform-android-sdk/dependencies)until next release.
